js/main.js

```diff
@@ -21,6 +21,68 @@ function setDisabledState(disabled, d) {
   target.attr("fill", color);
 }
 
+// interpret URL hash
+function setState() {
+  var statesSelected = Array();
+
+  // find the graph
+  nodes = document.getElementById('theImage')
```

Comment:
wait on `document.addEventListener("DOMContentLoaded", function() { }`? That might not work super well either because we're injecting the SVG later... alternatively call this function after we insert the SVG?
Minor suggestion, shrugging and saying it works OK like this is also OK